Deployments/suex-enginfprojects-test/k8s-manifests/deploy-mariadb.yml

```
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: enginfprojects-mariadb
+  namespace: suex-enginfprojects-test
+  labels:
+    app.kubernetes.io/name: enginfproects-mariadb
+    app.kubernetes.io/component: database
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: enginfprojects-mariadb
+  template:
+    metadata:
```

```suggestion
    metadata:
      annotations:
        k8up.io/backupcommand: mysqldump -uroot -pfixme --all-databases
```
To be used for the backup https://k8up.io/k8up/2.7/how-tos/application-aware-backups.html